src/mono/wasm/build/WasmApp.targets

```diff
@@ -135,6 +140,8 @@
 
     <!-- by default, package assemblies as webcil -->
     <WasmEnableWebcil Condition="'$(WasmEnableWebcil)' == ''">true</WasmEnableWebcil>
+
+    <WasmRuntimeAssetsLocation Condition="'$(WasmRuntimeAssetsLocation)' == ''">_framework</WasmRuntimeAssetsLocation>
```

Comment:
Thanks! I missed it, the _framework should be just a default value.

Reply:
Could we change advanced sample to use flat structure? Just to have at least some test.

Reply:
I had to introduce a new property `WasmEnableRuntimeAssetsLocation`, because there is no way to determine if the property is "undefined" or user set it to empty string

Reply:
`./` would also work as "empty" so that we don't have to do that. Perhaps you may need to convert it to `WasmEnableRuntimeAssetsLocation` internally.